Sources/Receipt/ReceiptDetailSectionData.swift

```diff
@@ -101,27 +101,27 @@ struct ReceiptDetailSectionFeeData: ReceiptDetailSectionData {
             let stringFee = receipt.fee,
             let fee = Double(stringFee),
             fee != 0.0,
-            let amount = Double(receipt.amount) else {
+            let amount = Double(receipt.amount!) else {
```

Comment:
never use ! to force unwrap, because making it optional means it might not have data. Doing force unwrap will crash the app. Always do if let amount = receipt.amount {
... }